docs/guides/how_to_guides/configuring_datasources/how_to_configure_sorting_in_data_connectors.rst

```diff
@@ -0,0 +1,1029 @@
+    )
+
+
+8. **Check your Validator**
+
+  You can check to see if the correct Batch was retrieved by checking the ``active_batch``'s ``batch_definition``.
+
+  .. code-block:: python
+
+    my_validator.active_batch.batch_definition
+
+  The expected output should show ``batch_identifiers`` corresponding to ``test_eee.csv``, namely ``"{'name': 'test', 'letter': 'eee'}"}``
+
+  .. code-block:: python
+
+    {'datasource_name': 'mydatasource', 'data_connector_name': 'my_data_connector', 'data_asset_name': 'my_reports', 'batch_identifiers': "{'name': 'test', 'letter': 'eee'}"}
+
+
+  You can also check that the first few lines of your Batch are what you expect by running:
+
+  .. code-block:: python
+
+    my_validator.active_batch.head()
+
+  Now that you have a Validator, you can use it to create Expectations or validate the data.
+
+
+.. _how_to_guides__how_to_sort_batches_NumericSorter:
+
+Configuring a Numeric Sorter
+----------------------------
+
+If you have the following ``numeric_example/`` directory in your filesystem, and you want to treat ``*.csv`` files as batches within the ``my_data_asset`` DataAsset:
+
+**Note** : In our example, the ``base_directory`` is set to ``../``. If we are running this Notebook in the same folder as Great Expectations home directory (ie ``great_expectations/``), GE will begin looking for the files in the parent directory.
+
+  .. code-block:: bash
+
+    numeric_example/test_111.csv
+    numeric_example/test_222.csv
+    numeric_example/test_333.csv
+    numeric_example/test_444.csv
+    numeric_example/test_555.csv
+
+
+1. **Load or create a DataContext**
+
+  .. code-block:: python
+
+    import great_expectations as ge
+    from great_expectations.cli.datasource import sanitize_yaml_and_save_datasource
+    from great_expectations.core.batch import BatchRequest
+
+    context = ge.get_context()
+
+2. **Configure a Datasource**
+
+  In the following configuration, a Datasource is configured with a ``PandasExecutionEngine`` and ``ConfiguredAssetFilesystemDataConnector``.
+  The DataConnector is configured with a single DataAsset named ``my_data_asset``. It has the ``base_directory`` set to ``numeric_example/``
+  and the regex ``pattern`` is set to capture two ``group_names``, ``name`` and ``number``. A ``NumericSorter`` is configured for the ``number`` capture group, which
+  captures the section of the file name that looks like : ``111``, and sorts the Batches in decreasing order.
+
+  .. code-block:: python
+
+    config = f"""
+        name: mydatasource
+        class_name: Datasource
+        execution_engine:
+            class_name: PandasExecutionEngine
+        data_connectors:
+          my_data_connector:
+              module_name: great_expectations.datasource.data_connector
+              class_name: ConfiguredAssetFilesystemDataConnector
+              glob_directive: "*.csv"
+              base_directory: /Users/work/Development/GE_data/lexicographic_example/
```

Comment:
It might make sense to change this path to something generic like `path/to/your/project/`

Reply:
I thought I took all these out ... thanks for spotting